### ISSUE TITLE
Certificate findings do not mention involved domain

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-coordination#1499]
**Describe the bug**
When a certificate is expired for some domain KAT can create a finding for this. However, the finding itself does not tell you which domain the certificate has expired for. The same holds for almost expiring certificates.

**To Reproduce**
1. Scan a website with expired certificate
2. Notice the finding does not contain the domain name, neither in the interface nor in .pdf report.

**Expected behavior**
Showing `KAT-CERTIFICATE-EXPIRED @ example.com (Let's Encrypt)` instead of leaving `example.com` out. I would rather see the involved domain first than the involved certificate issuer.

**Screenshots**
![image](https://github.com/minvws/nl-kat-coordination/assets/115991818/d578e0e1-0c21-43cb-b1f2-cc6ac725b24f)

![image](https://github.com/minvws/nl-kat-coordination/assets/115991818/84b75f7d-6481-4858-838a-a802f10a2f23)

**OpenKAT version**
1.10.0rc2

**Additional context**
It it possible to find out the domain name by clicking towards the certificate object, but this is not user friendly and not available in a report.

**Comment by praseodym** [comment]
I agree that the current view is not too useful. However, a single certificate can be valid for many domain names, and having tens or hundreds of domain names in the `subjectAltName` field is not uncommon. Which domain name should we show, all of them?

**Comment by stephanie0x00** [comment]
I'd suggest using the subject of the certificate, thus CN=example.com, als the Common Name field generally gives a good indication what type of system or specifically the domain is used.
